internal/sorter/file.go

```diff
@@ -0,0 +1,38 @@
+package sorter
+
+import (
+	"errors"
+	"io/ioutil"
+	"strings"
+)
+
+func getBlueBin() (map[string]bool, error) {
```

Comment:
`getRedBin` and `getBlueBin` can be the same function, pass in the file path as a parameter